templates/config.json

```diff
@@ -576,6 +609,12 @@
       "provider": "JupiterOne",
       "summary": "Metrics are defined to measure the effectiveness of controls and they are continuously monitored.",
       "guidance": "You will need to define a set of KPIs and metrics for monitoring and reporting on the effectiveness of the security controls. You can leverage JupiterOne's querying/reporting capability to automate the reporting process.",
+      "instructionsSummary": "Confirm/tailor security program success metrics, reports location, and scorecard frequency.",
+      "instructions": [
+        "What metrics will you use to measure the success of your security program?",
+        "How frequently will scorecards be produced?",
+        "Where will reports and scorecards be stored?"
```

Comment:
I think this is fine but perhaps not necessary

Reply:
Would you keep any of the above questions?